documentation/modules/deploying/con-deploy-cluster-operator-from-operator-hub.adoc

```diff
@@ -1,21 +1,26 @@
 // Module included in the following assemblies:
 //
-// deploying/assembly_deploy-cluster-operator.adoc
+// deploying.adoc
 
-[id='deploying-cluster-operator-from-operator-hub-{context}']
-= Deploying the Cluster Operator from OperatorHub.io
+[id='deploying-strimzi-from-operator-hub-{context}']
+= Deploying Strimzi from OperatorHub.io
 
 [role="_abstract"]
 {OperatorHub} is a catalog of Kubernetes Operators sourced from multiple providers.
-It offers you an alternative way to install stable versions of Strimzi using the Strimzi Kafka Operator.
+It offers you an alternative way to install stable versions of Strimzi using the *Strimzi Kafka* operator.
```

Comment:
This is explanation for new users who don't know OperatorHub. But I think it makes no sense ... you basically say you can _install Strimzi using Strimzi_. I think it would be better to say something more generic, e.g. _install stable version of Strimzi from its catalog of operators_ or something. Possibly adding somewhere in here also link to https://operatorhub.io/operator/strimzi-kafka-operator would help.

Reply:
I changed the attribute to go straight to the Strimzi page. And changed the text as follows:

> OperatorHub.io is a catalog of Kubernetes operators sourced from multiple providers.
> It offers you an alternative way to install a stable version of Strimzi.

The link is later in the text: 

> To install Strimzi, locate Strimzi from [OperatorHub.io](https://operatorhub.io/operator/strimzi-kafka-operator), and follow the instructions provided...